scriptmodules/supplementary/backends.sh

```diff
@@ -18,6 +18,9 @@ function _list_backends() {
     local id="$1"
     backends=()
 
+    # skip the 'depends' packages
+    [[ ${__mod_info[$id/section]} == "depends" ]] && return 1
+
```

Comment:
Which dependency packages were showing that need to be skipped ?

Reply:
I think `sdl1` showed up if `depends` were not skipped.

Reply:
It shouldn't - The "skip" is fine as we can avoid the extra logic but I would like to reproduce the issue to see what's going on. I just tested and couldn't.

Reply:
Hm, it does show up for me if (from this PR's branch) I remove the line in question. Do you have the `sdl1` module installed ?

Reply:
Ah I didn't use the branch as didn't notice the change to the sdl1 module flags (adding dispmanx to it). Makes sense now.